.github/workflows/ci-test.yml

```
@@ -0,0 +1,42 @@
+
+on:
+  push:
+    branches: 
+      - main
+  pull_request:
+    branches:
+      - main
+    types: [opened, synchronize]
+
+  jobs:
+    build:
```

actionlint

unexpected key "build" for "jobs" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)